### ISSUE TITLE
How do I get the files from the mod back into a functioning tmod file?

### ISSUE THREAD

**Issue by YakiiYukii** [IVogel/tModUnpacker#4]
I wanted to edit the textures from Terraria Overhaul for personal use, but I cant find any way to repack the files into a tmod file, is there another tool that does that or is there another way to do so?

**Comment by IVogel** [comment]
Actually, there is no such tool. (Also, maybe it's just that I don't know these tools.)
tModLoader packs the mod after it was built from sources.
Should I make mod packer? :P (Or, atleast try to do so)

**Comment by YakiiYukii** [comment]
Yeah if you made a tool that could do that it would be really cool! thanks for replying.

**Comment by P3rf3ctXZer0** [comment]
You could just make a mod XD Just call your project terraria overhaul Edit by 5hit PU
